test/initAction.d.ts

```diff
@@ -0,0 +1 @@
+export const initAction: { type: string };
```

Comment:
Is there a way to change this `string` to more specific one?
I'm afraid that this would mess up some union type like
```
// Following will be just a string
type actionType = (myAction1 | myAction2 | initAction)['type'];
```

Reply:
Oh, sorry. It's just for the test. My mistake.